cmd/commands/service.go

```diff
@@ -377,6 +377,7 @@ Additional curl arguments and flags may be specified after a double dash (--).`,
 		Args: UpToDashDash(ArgValidationConjunction(
 			cobra.MinimumNArgs(serviceInvokeMaxNumberOfArgs-1),
 			cobra.MaximumNArgs(serviceInvokeMaxNumberOfArgs),
+			// TODO validate that at most one content-type flag is set
```

Comment:
This is better done in `PreRunE` in our conventions, because `Args` is for ... args.
Will add